### DISCUSSION TITLE
Diskusi saran untuk html frontend

### DISCUSSION THREAD

**Discussion by iqbalmusyaffa** [semicolon-coder/html-titikoma-fe#2] [Ideas]
Kalau ada saran mengenai HTML frontend silahkan reply ini

**Comment by mharits-s** [comment]
![WhatsApp Image 2022-03-20 at 20 10 51](https://user-images.githubusercontent.com/100942834/159164024-2cb72c8e-c857-4387-80f3-58eff10b2941.jpeg)
Saya menemukan bug pada home page untuk mobile screen dan tambahkan favicon

**Reply by IrvanSN** [reply]
Okee, untuk bagian input email nanti akan saya perkecil lagi biar tidak keluar dari screennya. Untuk favicon akan segera saya tambahkan. Terima kasih!

**Comment by Nathanael1721** [comment]
1. card menu kurang button add to cart per menu
2. bagian menu ada revisi, terutama pada bagian pilihan makanan dan kalau bisa tampilan banner berupa carousel agar lebih menarik
3. menu testimonial ada revisi minor
4. menu promo belum ada
5. tambahan untuk beberapa bagian agar nanti bisa dibuat pop up page seperti pada bagian detail makanan, pesanan, dan checkout

**Reply by IrvanSN** [reply]
Card menu bagian mana?

**Reply by iniryan** [reply]
oke mas nanti saya tambahin

**Reply by iniryan** [reply]
yg bagian popular menu itu lo mas irvan, nanti saya tambahin biar pas

**Reply by Nathanael1721** [reply]
@iniryan nanti tolong dibantu jelaskan bagian cart menunya

**Reply by IrvanSN** [reply]
> yg bagian popular menu itu lo mas irvan, nanti saya tambahin biar pas

Okee, boleh nanti saya tambahkan buttonnya

**Reply by IrvanSN** [reply]
untuk carousel masih dibikinkan sama @iqbalmusyaffa . untuk pop up page sepertinya akan susah untuk dibuat karena waktu sudah mepet deadline.

**Reply by Nathanael1721** [reply]
oke siap, nanti kita diskusikan bersama lagi
